.github/workflows/_check_coverage.yml

```diff
@@ -8,8 +8,6 @@ on:
     secrets:
       gcs-cache-key:
         required: true
-      gcp-key:
```

Comment:
ah, yep - this is still required here for cov upload

same below

Reply:
Fixed now, I guess this is the part I could not test in staging.

Reply:
PTAL